### DISCUSSION TITLE
unresolved import: rocket::response::content::Html

### DISCUSSION THREAD

**Discussion by gfox1** [rwf2/Rocket#2348] [Questions] [answered]
Hi there, I'm trying to use `rocket::response::content::Html` and for some reason I'm getting an unresolved import when the cargo dependency version is set to `rocket = { version = "0.5.0-rc.1", features = ["json"] }`.

Here is more of the error I am getting from the compiler:

```
error[E0432]: unresolved import `rocket::response::content::Html`
  |
8 | use rocket::response::content::Html;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `Html` in `response::content`
```

Any direction on why this is happening would be great! I can't seem to find anything in the docs or other discussions about it.

**Comment by SergioBenitez** [comment, marked as answer]
This is called [`RawHtml`](https://api.rocket.rs/v0.5-rc/rocket/response/content/struct.RawHtml.html) in 0.5.

**Reply by gfox1** [reply]
Hi Sergio, thanks for the quick response! 

I guess this previous discussion (https://github.com/SergioBenitez/Rocket/discussions/1936) threw me off, where it says that it was called `Html` in version 0.5. 

But I do see in the docs now that it says `RawHtml`: https://api.rocket.rs/master/rocket/response/content/struct.RawHtml.html
